web/src/lib/components/shared-components/user-avatar.svelte

```diff
@@ -15,6 +16,13 @@
   export let showTitle = true;
   export let autoColor = false;
   let showFallback = true;
+  let appendix = '';
```

Comment:
this should really be named 'query', though given my earlier comment about cache control this code will probably be removed anyway